tests/basic/sharedv4_svc_test.go

```diff
@@ -259,84 +266,303 @@ var _ = Describe("{Shared4SvcFailoverIO}", func() {
 			}
 		})
 
-		Step("for each context, restart volume driver on NFS server node and verify I/O", func() {
-			for _, ctx := range testSv4Contexts {
+		for _, ctx := range testSv4Contexts {
+			Step(fmt.Sprintf("induce multiple sharedv4 service failovers and verify I/O for app %s", ctx.App.Key), func() {
 				vols, err := Inst().S.GetVolumes(ctx)
 				Expect(err).NotTo(HaveOccurred())
 
 				apiVol, err := Inst().V.InspectVolume(vols[0].ID)
 				Expect(err).NotTo(HaveOccurred())
 
 				// verify failover and failback by repeating the steps below
-				for i := 0; i < 2; i++ {
+				for i := 0; i < 4; i++ {
```

Comment:
nit: could this be configurable if trying to repro a certain case w/ a number of failovers? If that is too much overhead, not a problem - four sounds good.

Reply:
Yes, that might be useful. Changed to retrieve the number of failovers from an env variable.